src/components/NewConversationDialog/NewConversationDialog.vue

```diff
@@ -295,16 +302,25 @@ export default {
 			this.page = 2
 
 			try {
-				this.newConversation.token = await this.$store.dispatch('createGroupConversation', {
-					conversationName: this.conversationName,
-					isPublic: this.isPublic,
-				})
-
 				// Gather all secondary requests to run in parallel
 				const promises = []
 
-				if (this.isPublic && this.password && this.newConversation.hasPassword) {
-					promises.push(setConversationPassword(this.newConversation.token, this.password))
+				if (this.isPublic && this.forcePasswordProtection && this.supportConversationCreationPassword) {
```

Comment:
There should not be changes apart from passing `password: this.password`

`this.forcePasswordProtection && this.supportConversationCreationPassword` do not belong here, I think, first should block UX from sending a request, if truthy, and second - checked in store